hollow/src/main/java/com/netflix/hollow/api/codegen/HollowConsumerJavaFileGenerator.java

```diff
@@ -82,4 +136,23 @@ private String createFullPackageName(String packageName, String subPackageName,
     private boolean isEmpty(String value) {
         return value == null || value.trim().isEmpty();
     }
+
+    private void appendImportFromBasePackage(StringBuilder builder, String leaf) {
+        builder.append("import ").append(packageName).append(".").append(leaf)
+            .append(";\n");
```

Comment:
minor but it seems like your line wrap is a bit aggressive.  Perhaps in the future it would be nice to be a bit more lenient ;)

Reply:
yeah, I've been wrapping at 90 for a few years, a little more difficult with 4 space indents, though